src/test/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentTSnePlotControllerWIT.java

```diff
@@ -38,6 +42,8 @@
 @ContextConfiguration(classes = TestConfig.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class JsonExperimentTSnePlotControllerWIT {
+    private static final String URL = "/json/experiments/{experimentAccession}/metadata";
```

Comment:
Seeing as there are quite a few other tests in this class that use other URLs, could you change this to something a little less generic, e.g. `EXPERIMENT_METADATA_ENDPOINT_URL`?

Reply:
Done!